vnext/ReactUWP/Views/SliderViewManager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <Views/ShadowNodeBase.h>
+#include "SliderViewManager.h"
+
+#include <Utils/ValueUtils.h>
+
+#include <IReactInstance.h>
+
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+namespace winrt {
+using ToggleButton = Windows::UI::Xaml::Controls::Primitives::ToggleButton;
+}
+
+namespace react {
+namespace uwp {
+
+class SliderShadowNode : public ShadowNodeBase {
+  using Super = ShadowNodeBase;
+
+ public:
+  SliderShadowNode() = default;
+  void createView() override;
+  void updateProperties(const folly::dynamic &&props) override;
+};
+
+void SliderShadowNode::createView() {
+  Super::createView();
+
+  auto slider = GetView().as<winrt::Slider>();
+  auto wkinstance = GetViewManager()->GetReactInstance();
+}
+
+void SliderShadowNode::updateProperties(const folly::dynamic &&props) {
+  m_updating = true;
+  Super::updateProperties(std::move(props));
+  m_updating = false;
+}
+
+SliderViewManager::SliderViewManager(
+    const std::shared_ptr<IReactInstance> &reactInstance)
+    : Super(reactInstance) {}
+
+const char *SliderViewManager::GetName() const {
+  return "RCTSlider";
+}
+
+folly::dynamic SliderViewManager::GetNativeProps() const {
+  auto props = Super::GetNativeProps();
+
+  props.update(
+      folly::dynamic::object("value", "integer")("disabled", "boolean"));
+
+  return props;
+}
+
+facebook::react::ShadowNode *SliderViewManager::createShadow() const {
+  return new SliderShadowNode();
+}
+
+XamlView SliderViewManager::CreateViewCore(int64_t tag) {
+  auto slider = winrt::Slider();
+  slider.MinHeight(100);
+  return slider;
+}
+
+void SliderViewManager::UpdateProperties(
+    ShadowNodeBase *nodeToUpdate,
+    const folly::dynamic &reactDiffMap) {
+  auto slider = nodeToUpdate->GetView().as<winrt::Slider>();
+  if (slider == nullptr)
+    return;
```

Comment:
I don't think we need to do this check.

Reply:
I think we do, as and try_as return null if the QI fails

---
In reply to: [335128424](https://github.com/microsoft/react-native-windows/pull/3400#discussion_r335128424) [](ancestors = 335128424)